importer/importer/builder.py

```diff
@@ -958,12 +980,11 @@ def build_assign_payload(rows, resource_type, url_filter):
                     resource, item_id, organization_name
                 )
             if resource_type == "List":
-                resource = update_list(resource, item_id, supply_date)
+                resource = update_list(resource, subject_id, item_id, supply_date)
 
             if "meta" in resource:
                 version = resource["meta"]["versionId"]
                 resource_id = resource["id"]
-                del resource["meta"]
```

Comment:
Could you explain why this is no longer needed?

Reply:
Line `909-917` updates the meta to add the tags i.e. `REL` and `Location` tags.  This line would remove them everytime after they added.